core/src/test/scala/io/aiven/guardian/kafka/Generators.scala

```diff
@@ -19,12 +19,13 @@ object Generators {
                                    key: String,
                                    timestamp: Long
   ): Gen[ReducedConsumerRecord] = for {
-    t             <- Gen.const(topic)
-    p             <- Gen.const(0) // In mocks this value is irrelevant and in tests we always have a single partition
-    o             <- Gen.const(offset)
-    k             <- Gen.const(key)
-    value         <- Gen.alphaStr.map(string => Base64.getEncoder.encodeToString(string.getBytes))
-    ts            <- Gen.const(timestamp)
+    t         <- Gen.const(topic)
+    p         <- Gen.const(0) // In mocks this value is irrelevant and in tests we always have a single partition
+    o         <- Gen.const(offset)
+    k         <- Gen.const(key)
+    valueSize <- Gen.choose(1, 100)
+    value <- Gen.stringOfN(valueSize, Gen.alphaChar).map(string => Base64.getEncoder.encodeToString(string.getBytes))
```

Comment:
Just for info -- is this stringOfN always valueSize, or "up to" valueSize?  The ScalaCheck doc is a bit vague!

Reply:
Its always `valueSize`. The `valueSize <- Gen.choose(1, 100)` is actually what specifies.